### ISSUE TITLE
Change in subtemplate pcc:bf2:PublicationInformation that affects conversion was:Find a way to be able to either legally use literals where there should be URIs

### ISSUE THREAD

**Issue by NancyL** [LD4P/rdf2marc#209]
**Type of change**
 A mapping I need to change in the template I believe affects conversion

**Example RDF records**
Please provide URIs or RDF of the `bf:Instance`. If providing RDF, Turtle is preferred.
https://api.stage.sinopia.io/resource/d7bb311f-a7d8-463e-8537-d71c76bb2ee5

**Resource template**
Which resource template does this apply to? In which Sinopia environment (development, stage, production)?
pcc:bf2:Monograph:Instance; nested template pcc:bf2:PublicationInformation

I need to change the bf:place property from a lookup to a nested resource. In my own experimentation, this seems to create an error in conversion to MARC

**Expected mapping logic**
Please describe the logic of the mapping.

bf:place <URI> ;
   <URI> [ a bf:Place ;
    rdfs:label "New Sage"
] .

or if no URI available:
bf:place [a bf:Place ;
rdfs:label "New Sage"
].

**Comment by justinlittman** [comment]
@NancyL Here is what I get for https://api.stage.sinopia.io/resource/d7bb311f-a7d8-463e-8537-d71c76bb2ee5:
```
LEADER      nam a22     uu 4500
008 220606s2017    xx                  eng||
020    $a 9780393609097
040    $a cst $b eng $c cst $e rda
050  0 $a BL860 $b .G35 2017
100 1  $a Blanco, Paca $e Author $0 https://api.stage.sinopia.io/resource/cbc90c2f-a8ce-4e18-b18a-5c790a508f66
245 10 $a Norse mythology $c Neil Gaiman
264  1 $b Aristolochia $c 2017
300    $a 293 pages $c 24 cm
336    $a text $b txt $0 http://id.loc.gov/vocabulary/contentTypes/txt $2 rdacontent
337    $a unmediated $b n $0 https://id.loc.gov/vocabulary/mediaTypes/n $2 rdamedia
338    $a volume $b nc $0 https://id.loc.gov/vocabulary/carriers/nc $2 rdacarrier
500    $a Transcribed publication statement: New York ; London : W.W. Norton & Company, [2017]
650  7 $a Mythology, Norse $2 fast $0 http://id.worldcat.org/fast/1031869
655  0 $a Myths $0 http://id.loc.gov/authorities/genreForms/gf2015026043 $2 lcgft
758    $4 http://id.loc.gov/ontologies/bibframe/instanceOf $i Instance of: $a Norse mythology $0 https://api.stage.sinopia.io/resource/8dcb039b-5505-4645-b2a5-a21498d3c0bd
884    $a Sinopia rdf2marc $g 20220725 $k https://api.stage.sinopia.io/resource/d7bb311f-a7d8-463e-8537-d71c76bb2ee5 $u https://github.com/LD4P/rdf2marc
```

What is expected?

**Comment by NancyL** [comment]
So actually this example didn't have a place of publication in the data. 
But there was 2 parts to this problem:
1. I needed to change bf:place here to go to a nested resource rather than a direct lookup and thought it might affect conversion. Turns out it does not.
2. We need to be able to put literals in this field (which would be reified by the addition of the bf:Place range). I thought adding the nested resource would resolve this, but it turns out it does not and I finally realize why. It is because the conversion does the lookup to LC-NAF and doesn't find anything. At least it gets this error message:
![image](https://user-images.githubusercontent.com/5078198/181824831-9df4ac71-e436-4be9-a3af-2361779db3a0.png)
So the same issue I think as we had with new NARs when we tried to enter them.

However. LC wrote me and is saying they might completely change how they do provision activity, so perhaps best not to spend more time on this until we know for sure they are changing.
